timemachine/cpp/src/context.cu

```diff
@@ -46,6 +48,16 @@ Context::Context(
 
 };
 
+Context::Context(
+    int N,
+    const double *x_0,
+    const double *v_0,
+    const double *box_0,
+    Integrator* intg,
+    std::vector<BoundPotential *> bps) :
+    Context(N, x_0, v_0, box_0, intg, nullptr, bps) {};
```

Comment:
Did you intend to disable the definition? Invoking this constructor would instantiate an invalid object since the internals aren't initialized.

Reply:
I did not. Was trying to use a delegated constructor, but sounds like I borked it